<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_recipe</class>
 <widget class="QWidget" name="new_recipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新建食谱</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="choose_food_button">
       <property name="text">
        <string>选择食物</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_file_button">
       <property name="text">
        <string>从文件导入...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="first_page_button">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
